mod_info.json

```diff
@@ -4,7 +4,12 @@
 	"author":"Originem",
 	"version":"1.3",
 	"description":"*Requires LazyLib \nA religious group which believe in nature.",
-	"gameVersion":"0.9.1a",
+	"gameVersion":"0.95a-RC15",
 	"jars":["jars/CMC.jar"],
+    "dependencies": [
```

Comment:
Makes the game not start without these. Excluded Nexerelin, as that is optional for CMC.